packages/stream-metadata/src/routes/spaceImage.ts

```diff
@@ -18,6 +18,12 @@ const paramsSchema = z.object({
 		}),
 })
 
+const CACHE_CONTROL = {
+	307: 'public, max-age=30, s-maxage=300',
+	'4xx': 'public, max-age=30, s-maxage=3600',
+	'5xx': 'public, max-age=30, s-maxage=300', // TODO: should we cache the 500s?
```

Comment:
In other code we turn the 500's to 400's. I believe 500's should be limited to code and systems outside of our control so we can differentiate quickly if we have logical issue or a systems issue.

Reply:
@brianathere agreed, just turned it into a 422